Library/Source/NativeEngine.h

```diff
@@ -384,6 +390,7 @@ namespace Babylon
         bx::DefaultAllocator m_allocator;
         uint64_t m_engineState;
 
+        inline static BGFXCallback m_bgfxCallback{};
```

Comment:
Does this need to be static?

Reply:
yes, I need it in void NativeEngine::InitializeWindow. Which is static.

Reply:
Okay. Probably fine for now, then; but long-term, I think we should move to having as little static state as possible. Maybe `NativeEngine::InitializeWindow` could return a state object, for example, which could hold onto this and have its lifecycle tied to the JavaScript context, or something like that.